pkg/quay/quay.go

```diff
@@ -275,3 +275,79 @@ func (c *QuayClient) AddPermissionsToRobotAccount(organization, imageRepository,
 	fmt.Println(string(body))
 	return nil
 }
+
+// Returns all repositories of the DEFAULT_QUAY_ORG organization (used in e2e-tests)
+func (c *QuayClient) GetAllRepositories(organization string) ([]Repository, error) {
+	url := fmt.Sprintf("%s/repository?last_modified=true&namespace=%s", c.url, organization)
+
+	req, err := http.NewRequest(http.MethodGet, url, nil)
+	if err != nil {
+		return nil, err
+	}
+
+	req.Header.Add("Authorization", fmt.Sprintf("%s %s", "Bearer", c.AuthToken))
+	req.Header.Add("Content-Type", "application/json")
+
+	res, err := c.httpClient.Do(req)
+	if err != nil {
+		return nil, err
+	}
+	if res.StatusCode != 200 {
+		fmt.Printf("error getting repositories, got status code %d", res.StatusCode)
```

Comment:
should this message be added to the error returned below? (i.e. wrapped by `fmt.Errorf()`

and similarly in [the other method](https://github.com/redhat-appstudio/image-controller/pull/19/files#diff-d4a36e5f4e14c5636dfed4031b618cc2a45afc79934eabb0336a1adde773fd1bR334) as well